local/test-binary

```diff
@@ -15,14 +15,26 @@ set -o errexit
 set -o xtrace
 
 WORKDIR_ABS=$(cd ${1:-./build}; pwd -P)
+rm -fr ${WORKDIR_ABS}/PS
 mkdir -p ${WORKDIR_ABS}/PS
 tar -C ${WORKDIR_ABS}/PS --strip-components=1 -zxpf $(ls $WORKDIR_ABS/*.tar.gz | head -1)
 cd ${WORKDIR_ABS}/PS/mysql-test
-
+#
+# CentOS 6 & 7
+if [[ -f /opt/rh/devtoolset-7/enable ]]; then
+        source /opt/rh/devtoolset-7/enable
+fi
+#
 TOKUDB_PLUGIN=$(find $WORKDIR_ABS -type f -name 'ha_tokudb.so')
-HOTBACKUP_LIB=$(find $WORKDIR_ABS -type f -name 'libHotBackup.so')
-HOTBACKUP_PLUGIN=$(find $WORKDIR_ABS -type f -name 'tokudb_backup.so')
+HOTBACKUP_LIB=$(find $WORKDIR_ABS -type f -name 'libHotBackup.so' | head -n1)
+HOTBACKUP_PLUGIN=$(find $WORKDIR_ABS -type f -name 'tokudb_backup.so'| head -n1)
```

Comment:
Why?

Reply:
pls see above

Reply:
See my reply above, this looks almost-broken

Reply:
currently it's 
tar -C ${WORKDIR_ABS}/PS --strip-components=1 -zxpf $(ls $WORKDIR_ABS/*.tar.gz | head -1)
WORKDIR_ABS contains all files, even garbage, but ${WORKDIR_ABS}/PS contains only unpacked tarball

Reply:
is not needed anymore